extensions/matsim2silo/src/main/java/de/tum/bgu/msm/matsim/StandaloneSkimCreator.java

```diff
@@ -28,37 +30,59 @@
 
 public class StandaloneSkimCreator {
 
+    private final static Logger logger = Logger.getLogger(StandaloneSkimCreator.class);
+
     public static void main(String[] args) {
-        createSkims(Integer.parseInt(args[0]), args[1], args[2], args[3], args[4], Integer.parseInt(args[5]));
+//        createSkims(Integer.parseInt(args[0]), args[1], args[2], args[3], args[4], Integer.parseInt(args[5]));
+        createSkims(8, "C:\\Users\\Nico\\tum\\fabilut\\gitproject\\muc\\input\\zonesShapefile\\zones.shp",
+                "C:\\Users\\Nico\\tum\\fabilut\\gitproject\\muc\\input\\mito\\trafficAssignment\\pt_2020\\network_pt_road.xml.gz",
+                "C:\\Users\\Nico\\tum\\fabilut\\gitproject\\muc\\input\\mito\\trafficAssignment\\pt_2020\\schedule.xml",
+                "skimout.omx", 28800);
     }
 
     public static void createSkims(int threads, String shapePath, String networkPath,
                                    String schedulePath, String outputPath, int time) {
 
+        SiloUtil.loadHdf5Lib();
+
+        logger.info("Preparing config...");
         Config config = ConfigUtils.createConfig();
+        config.transit().setUseTransit(true);
 
+        logger.info("Reading network...");
         Network network = NetworkUtils.createNetwork();
         new MatsimNetworkReader(network).readFile(networkPath);
 
         Scenario scenario = ScenarioUtils.createScenario(config);
+        logger.info("Reading schedule...");
         new TransitScheduleReader(scenario).readFile(schedulePath);
         TransitSchedule schedule = scenario.getTransitSchedule();
 
+        logger.info("Reading shape file...");
         final Collection<SimpleFeature> features = ShapeFileReader.getAllFeatures(shapePath);
```

Comment:
I suppose error occurs when reading the shape file. here is the error message
https://gist.github.com/key2000/f8f55f5a412624fe72bab17967f4013d

Reply:
Could you try again with the master version now? It runs on my machine :)

Reply:
Thank you for quick modification, but I have got same error in reading shape file.

https://gist.github.com/key2000/04cc80a8f5d593995b2d0bcacdda025c